.github/workflows/python-checks.yml

```diff
@@ -32,6 +32,12 @@ jobs:
       - name: Install dependencies
         working-directory: flink-ml-python/dev
         run: python -m pip install -r dev-requirements.txt
+      - name: Build Python package
+        working-directory: flink-ml-python
+        run: python setup.py sdist
+      - name: Install Python package
+        working-directory: flink-ml-python
+        run: python -m pip install dist/*.tar.gz
```

Comment:
I think we can create a dedicated workflow action to build python package and this can be used in the release.

Reply:
I found we forgot to add license header to `java8-build.yml` and `python-checks.yml`

Reply:
I have added the license header to Github Action config files.

About creating a workflow to build the python package, I think there is no need to do so now, given that the workload to build the package on a local machine is acceptable so far, and the process is not complicated.

Reply:
I have added the license header to Github Action config files.

About creating a workflow to build the python package, I think there is no need to do so now, given that the workload to build the package on a local machine is acceptable so far, and the process is not complicated.